metadata_backend/helpers/schemas/bp_bpsample.json

```diff
@@ -55,7 +55,7 @@
                     "title": "Tag title"
                 },
                 "value": {
-                    "type": "string",
+                    "type": ["string", "integer"],
```

Comment:
having values like this might be difficult for front-end to render forms from json, that is if we are rendering forms from it.

if we do this for one specific attribute then we might need to handle all, could we have more information on why we need to do this type conversion?

another thing is that give that the attribute field also takes Units, consider some units might be float in value , so `number` type might be better

Reply:
I'm not 100% sure on the necessity of this either but it was easy to change. The context provided for this is in the issue this PR is related to:
```
Additional context

This is needed to simplify queries in bp beacon.
```

Reply:
And good point about the `number` type

Reply:
> This is needed to simplify queries in bp beacon.

yeah, not sure this should be fixed on this, as it will affect the form generation in the front-end, and as I mentioned if we do this for one value we would need to do it for all attributes.

I have some workaround solution where we don't modify the schema, but instead when we insert in the database we add an optional parameter to the attribute e.g. `attributeDataType` which specifies how that attribute should be interpreted by an a service that gets data from metadata submitter, and in that way we will try to infer the data instead of having special cases for each attribute, and if we cannot infer it we default to string

Reply:
the `attributeDataType` does not need to be present in the json schema, but instead we can add it to the XML/JSON processing of the data whenever it is posted, and will be specific for `attribute` fields

Reply:
better yet instead of us inferring the data type we should ask the XML schema provider to specify it, as this alleviate us trying to determine which data type each attribute is.

Reply:
Ok we now discovered that the XML schemas provided to us also specify numeric attributes as decimals and this `age_at_extraction` tag should go within that and not under a regular attribute (which the current test files some reason didn't do and just parsed the value into string, hence the confusion). Anyway, the parser parses the numeric attribute into a float already as is and hence this PR is now pointless.